### PR TITLE
Feat/Update image/copy in WritePermissionDialog

### PR DESCRIPTION
**Pull request by KryptKode**
## Notes
- rename function to execute after SAF action on SDK 30 from `funAfterDelete30File` to `funAfterSdk30Action`
- image / copy changes to `WritePermissionDialog`
- add missing method `isShowingSAFCreateDocumentDialogSdk30`
- extract `android.content.extra.SHOW_ADVANCED` to a constant `EXTRA_SHOW_ADVANCED` as it is used in many locations
### Screenshots
**Image** | **Video**
---|---
Screenshot | <video src="https://user-images.githubusercontent.com/25648077/158629893-5a1557e2-f92f-4cbe-9df4-9048aa806c3a.mp4" width="320"/>

### CODE REVIEW

File: commons/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -196,6 +196,7 @@
     <string name="confirm_storage_access_text_sd">If you don\'t see the SD card, try this</string>
     <string name="confirm_storage_access_android_text">Please allow the app accessing the selected storage on the next screen by pressing \'Use this folder\' at the bottom.</string>
     <string name="confirm_storage_access_android_text_specific">Please allow accessing \'%s\' on the next screen by pressing \'Use this folder\' at the bottom.</string>
+    <string name="confirm_create_doc_for_new_folder_text">Please press \'Save\' at the bottom on the next screen to create the new folder</string>
```
Comment (tibbi) [inline commons/src/main/res/values/strings.xml:199]: please add a dot in the end and change "on the next screen" to "of the next screen"

**Comment by tibbi** [comment]
is this needed at creating new folders at SD card only? I dont see any other app requesting it at the internal storage. They juse use the old "Use this folder" permission.

**Comment by KryptKode** [comment]
From my observation, we cannot create a folder on the root of both internal storage and SD Card without using `ACTION_CREATE_DOCUMENT`. 

Can you share links to the apps in question so I can check?

**Comment by tibbi** [comment]
ah, this is only about the root folders? I was testing casual subfolders

**Comment by KryptKode** [comment]
It's about the root folder.

**Comment by tibbi** [comment]
okay, I cannot even reproduce that with other apps :) Either it is not possible, or it just defaults to DCIM. Guess they didnt manage implementing it.

**Comment by KryptKode** [comment]
Yes, most gallery apps create folders relative to DCIM. Users cannot select the path outside of this

File: commons/src/main/res/values/strings.xml (hunk 1 above)
Comment (KryptKode) [reply commons/src/main/res/values/strings.xml]: Done

**Comment by tibbi** [comment]
looks good, thanks. Will create the screenshots tomorrow

**Pull request merged**